### PR TITLE
Add cn.navclub.ldbfx

### PR DESCRIPTION
**Pull request by GZYangKui**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

**Comment by GZYangKui** [comment]
I tested my project locally through appstream-compose, and there was no missing description tag locally
appstream-compose

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

**Comment by GZYangKui** [comment]
bot, stop cn.navclub.ldbfx

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

File: cn.navclub.ldbfx.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hfiguiere) [inline cn.navclub.ldbfx.appdata.xml:1]: I mentionned this in the PR you close and thus it got ignored. There is an appdata file in the upstream repo that is installed. So you don't need that one here.

It's never installed anyway.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: cn.navclub.ldbfx.appdata.xml (hunk 1 above)
Comment (GZYangKui) [reply cn.navclub.ldbfx.appdata.xml:1]: OK, I'll remove the appdata

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

File: cn.navclub.ldbfx.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "mariadb",
```
Comment (barthalion) [inline cn.navclub.ldbfx.json:17]: You should clean up unneeded binaries, headers, static libraries and man pages.

File: cn.navclub.ldbfx.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "ln -s /app/lib/mariadb/libmariadb.so /app/lib/libmariadb.so.3"
```
Comment (barthalion) [inline cn.navclub.ldbfx.json:20]: This doesn't feel right, and is prone to break, given your app is dynamically linked to whatever SONAME version the connector provides. I would change it to just move files matching `libmariadb.so*` to /app/lib.

File: cn.navclub.ldbfx.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "ldbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
```
Comment (barthalion) [inline cn.navclub.ldbfx.json:13]: Not a blocker, but consider using GtkFileChooserNative so your app uses file portal for accessing sqlite files.

**Comment by GZYangKui** [comment]
bot, build cn.navclub.ldbfx

**Pull request closed without merging**